packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.jsx

```diff
@@ -26,6 +26,15 @@ const validate = (errors, values) => (fn, arr, err_msg) => {
     });
 };
 
+const getLocation = (location_list, value, type) => {
```

Comment:
I think this one could be moved to shared and shared between account and core? as we're duplicating it.